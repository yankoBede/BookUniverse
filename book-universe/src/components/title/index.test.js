import React from 'react'
import renderer from 'react-test-renderer'
import Title from './index'

describe('Header Component', () => {
    it('should render authenticated routes', () => {
        const tree = renderer.create(
            <Title title="My Test"/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})