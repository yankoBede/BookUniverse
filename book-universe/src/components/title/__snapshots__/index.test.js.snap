// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render authenticated routes 1`] = `
<h1
  className="title"
>
  My Test
</h1>
`;
